README.md

```diff
@@ -12,6 +12,30 @@ All features are contributed by you, our amazing .NET community, and maintained
 
 And – the best part – the features you add to the .NET MAUI Toolkit may one day be included into the official .NET MAUI library! We leverage the Community Toolkits to debut new features and work closely with the .NET MAUI engineering team to nominate features for promotion.
 
+## Getting Started
+
+In order to use the .NET MAUI Community Toolkit you need to call the extension method in your `MauiProgram.cs` file as follows:
+
+```csharp
+public static MauiApp CreateMauiApp()
+{
+    var builder = MauiApp.CreateBuilder();
+
+    // Initialise the toolkit
+	builder.UseMauiApp<App>().UseMauiCommunityToolkit();
+
+    // the rest of your logic...
+}
```

Comment:
What do you think about expanding this a bit to add the namespace?

```suggestion
using CommunityToolkit.Maui;

public static class MauiProgram
{
	public static MauiApp CreateMauiApp()
	{
		var builder = MauiApp.CreateBuilder();
		
		// Initialise the toolkit
		builder.UseMauiApp<App>().UseMauiCommunityToolkit();

    // the rest of your logic...
    }
}
```

Reply:
Another option (and maybe a better option to improve intellisense discoverability), would be to change the namespace of `AppBuilderExtensions.shared.cs` to match `Microsoft.Maui.Hosting.MauiApp`.

This way, we wouldn't need to force the use to manually add `using CommunityToolkit.Maui` which would improve the onboarding experience.

```cs
namespace Microsoft.Maui.Hosting;

/// <summary>
/// Extensions for MauiAppBuilder
/// </summary>
public static class AppBuilderExtensions
{
	/// <summary>
	/// Initializes the .NET Maui Community Toolkit Library
	/// </summary>
	/// <param name="builder"><see cref="MauiAppBuilder"/> generated by <see cref="MauiApp"/> </param>
	/// <returns><see cref="MauiAppBuilder"/> initialized for <see cref="CommunityToolkit.Maui"/></returns>
	public static MauiAppBuilder UseMauiCommunityToolkit(this MauiAppBuilder builder)
	{
		return builder.UseMauiCommunityToolkitCore();
	}
}
```

Reply:
We'd want to update `CommunityToolkit.Maui.Core.AppBuilderExtensions` too:

```cs
namespace Microsoft.Maui.Hosting;

/// <summary>
/// <see cref="MauiAppBuilder"/> Extensions
/// </summary>
public static class AppBuilderExtensions
{
	/// <summary>
	/// Initializes the .NET MAUI Community Toolkit Core Library
	/// </summary>
	/// <param name="builder"><see cref="MauiAppBuilder"/> generated by <see cref="MauiApp"/> </param>
	/// <returns><see cref="MauiAppBuilder"/> initialized for <see cref="CommunityToolkit.Maui.Core"/></returns>
	public static MauiAppBuilder UseMauiCommunityToolkitCore(this MauiAppBuilder builder)
	{
		return builder;
	}
}
```

Reply:
I don't know which I prefer. I like the idea of making it more discoverable for sure!

Is it good practise to ship something like this which exposes something under another namespace? (I'm asking as I really don't know 😄 )

Reply:
> Is it good practise to ship something like this which exposes something under another namespace

It is a thing; I've seen it done before. And I think we're in a unique situation where it does make sense for us since we're so closely linked to the .NET MAUI team. 

Maybe we keep the current `namespace CommunityToolkit.Maui` for v1.0. Because changing it to `namespace Microsoft.Maui.Hosting` would not be a breaking change. But changing `namespace Microsoft.Maui.Hosting` back to `namespace CommunityToolkit.Maui` _is_ a breaking change.

Reply:
You make a good point re: being closely linked to the other team and therefore that namespace. I do like the pragmatic decision you have put forward so I am happy to proceed down that route 👍

Reply:
Let's move forward keeping `namespace CommunityToolkit.Maui` and table the change to `namespace Microsoft.Maui.Hosting` for a later date.

I'm sure the .NET MAUI Engineering team would want to weigh in first.

And if this isn't a best practice, we certainly don't want to start a trend for other devs to hijack `namespace Microsoft.Maui.Hosting` as well.